tools/bin/ecflow/processes/main/meteorological/get_med_metfcst.py

```diff
@@ -0,0 +1,194 @@
+    # global attributes
+    met_ds.attrs['author'] = ('John Abatzoglou - University of ' +
+                              'Idaho, jabatzoglou@uidaho.edu')
+    met_ds.attrs['date'] = datetime.now().strftime('%d %B %Y')
+    met_ds.attrs['note1'] = ('The projection information for this ' +
+                             'file is: GCS WGS 1984.')
+    met_ds.attrs['note2'] = ('These data were created using netCDF version 4.')
+    met_ds.attrs['note3'] = ('Days correspond approximately to calendar ' +
+                             'days ending at midnight, Mountain Standard ' +
+                             'Time (7 UTC the next calendar day)')
+    met_ds.attrs['note4'] = ('Bias corrected CFSV2 forecasts using the 4 ' +
+                             'most recent CFSV2 runs each day, downscaled ' +
+                             'to the gridMET (Abatzoglou, 2013) data.')
+    # latitude attributes
+    met_ds.lat.attrs['units'] = "degrees_north"
+    met_ds.lat.attrs['description'] = "latitude"
+    # longitude attributes
+    met_ds.lon.attrs['units'] = "degrees_east"
+    met_ds.lon.attrs['description'] = "longitude"
+    # time attributes
+    met_ds.time.attrs['units'] = "days since 1900-01-01 00:00:00"
+    met_ds.time.attrs['calendar'] = "gregorian"
+    met_ds.time.attrs['description'] = "days since 1900-01-01"
+    # parameter attributes
+    # precipitation
+    met_ds.precipitation_amount.attrs['units'] = "mm"
+    met_ds.precipitation_amount.attrs['description'] = ('Daily Accumulated' +
+                                                        ' Precipitation')
+    met_ds.precipitation_amount.attrs['cell_methods'] = ('time: sum(' +
+                                                         'intervals: 24 hours)')
+    # temperature
+    met_ds.tmmn.attrs['description'] = "Daily Minimum Temperature"
+    met_ds.tmmx.attrs['cell_methods'] = "time: minimum(interval: 24 hours)"
+    met_ds.tmmx.attrs['description'] = "Daily Maximum Temperature"
+    met_ds.tmmx.attrs['cell_methods'] = "time: maximum(interval: 24 hours)"
+    for var in ['tmmn', 'tmmx']:
+        met_ds[var].attrs['units'] = "degC"
+        met_ds[var].attrs['height'] = "2 m"
+    # wind speed
+    met_ds.wind_speed.attrs['units'] = "m/s"
+    met_ds.wind_speed.attrs['description'] = "Daily Mean Wind Speed"
+    met_ds.wind_speed.attrs['height'] = "10 m"
+    # shortwave radiation
+    met_ds.surface_downwelling_shortwave_flux_in_air.attrs['units'] = "W m-2"
+    met_ds.surface_downwelling_shortwave_flux_in_air.attrs['description'] = (
+        'Daily Mean Downward Shortwave Radiation At Surface')
+    # specific humidity
+    met_ds.specific_humidity.attrs['units'] = "kg/kg"
+    met_ds.specific_humidity.attrs['description'] = "Daily Mean Specific " + \
+        "Humidity"
+    met_ds.specific_humidity.attrs['height'] = "2 m"
+    return met_ds
+
+
+def main():
+    ''' Download meteorological forecast data for 90-day forecast
```

Comment:
Standardize on the docstring formatting? (this one doesn't match the quote marks from lines 26-27)